### PR TITLE
Display conditions in the pipeline describe command

### PR DESCRIPTION
**Pull request by chetan-rns**
# Changes

This PR will add `CONDITIONS` field to the Tasks section of the `pipeline desc` command and display the conditions associated with each pipeline task. Added a function for formatting conditions.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Run the code checkers with `make check`
- [x] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes
```
NA
```

### CODE REVIEW

File: pkg/formatted/conditions_test.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
+// Copyright © 2019 The Tekton Authors.
```
Comment (danielhelfand) [inline pkg/formatted/conditions_test.go:1]:
```suggestion
// Copyright © 2020 The Tekton Authors.
```

File: pkg/formatted/conditions_test.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+)
+
+func TestFormatTaskConditions(t *testing.T) {
+	tests := []struct {
+		desc       string
+		conditions []v1beta1.PipelineTaskCondition
+		want       string
+	}{
+		{
+			"task with no condtions",
```
[the remark this answers is not in the document]
Comment (chetan-rns) [reply pkg/formatted/conditions_test.go]: I've added the property names

**Pull request merged**